packages/stockflux-watchlist/src/components/watchlist/Watchlist.js

```diff
@@ -0,0 +1,262 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Should this file still be in the PR? it doesn't get referenced anymore, due to the refactor into app.js

Reply:
Yeah, they can go! Thanks!